app/models/public_body.rb

```diff
@@ -217,15 +217,19 @@ def set_api_key!
   end
 
   def self.find_by_name(name)
-    PublicBody.joins(:translations).
+    found = PublicBody.joins(:translations).
       where("public_body_translations.name = ?", name).
        first
+
+    PublicBody.find(found.id)
```

Comment:
So, what's happening here? Why are the translations not getting loaded through `where`?